drivers/mongo/example_test.go

```diff
@@ -24,8 +24,9 @@ func Example() {
 	ctx := context.Background()
 
 	client, err := mongo.Connect(ctx, options.Client().
-		ApplyURI("mongodb://127.0.0.1:27017"))
+		ApplyURI("mongodb://127.0.0.1:27017/?directConnection=true"))
 	checkErr(err)
+	defer client.Disconnect(ctx)
```

Comment:
// TODO

If goleaks show problems, github actions should fail. Now actions is pass even if goleaks alert about a problem.